### DISCUSSION TITLE
Find all issues for which there are no commit references

### DISCUSSION THREAD

**Discussion by johnrwolfe** [community/community#23323] [New to GitHub]
<p>How can I find all open issues within a repository for which there are no commit references?  When an issue is referenced in a commit message, the hash for that commit is stored in the issue.  What is the search syntax for finding issues that have not been referenced by a commit?</p>

**Comment by arahimh** [comment, marked as answer]
<p>Hey johnrwolfe, </p>
<p>Thank you for taking the time to write in to our Community! </p>
<p>&gt; What is the search syntax for finding issues that have not been referenced by a commit?</p>
<p>The short answer is, this is currently not a feature that GitHub supports. Our Product team welcomes feature requests and would love to hear more about your use case here:  <a href="https://support.github.com/contact/feedback" rel="nofollow noopener">https://support.github.com/contact/feedback</a>.</p>
<p>You can however, search for issues and pull requests on GitHub and narrow the results using specific search qualifiers in any combination. Please see our documentation here, <a href="https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests" rel="nofollow noopener">https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests</a>. </p>
<p>I would like suggest that there is a qualifier for searching by commit SHA: <a href="https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-by-commit-sha" rel="nofollow noopener">https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-by-commit-sha</a>. </p>
<p>This particular search qualifier focuses on the linking between issues and pull requests which could be helpful.</p>
<p>I hope this helps, and please let me know if you have any additional questions. </p>
